stixcore/idb/manager.py

```diff
@@ -225,6 +226,39 @@ def convert_mib_2_sqlite(*, in_folder, out_file, version_label):
                 cur.execute("insert into IDB (creation_datetime, version) "
                             "values (current_timestamp, ?);", (version_label,))
 
+                # inject custom calibrations
+
+                nextID = 0
+                for calibN, in cur.execute("select distinct PCF_CURTX from PCF " +
+                                           "where PCF_CURTX not NULL")\
+                                  .fetchall():
+                    nr = int(re.match(r'([a-z]+)([0-9]+)([a-z]+)', calibN, re.IGNORECASE).group(2))
+                    if nr > nextID:
+                        nextID = nr
+                nextID += 1
+
+                # NIX00405
+                nix = 'NIX00405'
+                count,  = cur.execute("select count(*) from PCF where PCF_NAME = ? " +
+                                      "AND PCF_CURTX not NULL", (nix,))\
+                             .fetchone()
+                if count == 0:
+                    pname, nextID = IDB.generate_calibration_name("CIX", nextID)
+                    cur.execute('''update PCF set
+                                    PCF_CURTX = ?,
+                                    PCF_CATEG = 'N',
+                                    PCF_UNIT = 's'
```

Comment:
is there a format here like `VOLT` should this be `sec` or just `s`?

Reply:
this part is not really flexible at the moment will wait for  https://github.com/i4Ds/STIXCore/issues/59 to clean it up